--- vmhgfs-only/inode.c	2015-01-07 15:51:14.000000000 -0800
+++ vmhgfs-only/inode.c.new	2015-03-06 07:04:18.000000000 -0800
@@ -1922,7 +1922,7 @@
                            p,
 #endif
                            &inode->i_dentry,
-                           d_alias) {
+                           d_u.d_alias) {
          int dcount = hgfs_d_count(dentry);
          if (dcount) {
             LOG(4, ("Found %s %d \n", dentry->d_name.name, dcount));
@@ -1975,7 +1975,7 @@
       /* Find a dentry with valid d_count. Refer bug 587879. */
       list_for_each(pos, &inode->i_dentry) {
          int dcount;
-         struct dentry *dentry = list_entry(pos, struct dentry, d_alias);
+         struct dentry *dentry = list_entry(pos, struct dentry, d_u.d_alias);
          dcount = hgfs_d_count(dentry);
          if (dcount) {
             LOG(4, ("Found %s %d \n", (dentry)->d_name.name, dcount));